base/bucket_gocb.go

```diff
@@ -1235,6 +1249,36 @@ func (bucket *CouchbaseBucketGoCB) UpdateXattr(k string, xattrKey string, exp ui
 	err, cas = RetryLoopCas("UpdateXattr", worker, bucket.Spec.RetrySleeper())
 	if err != nil {
 		err = pkgerrors.Wrapf(err, "Error during UpdateXattr with key %v", UD(k).Redact())
+		return cas, err
+	}
+
+	if isDelete && !supportsTombstoneCreation && makeDocCalled {
```

Comment:
Can you add a comment above this explaining what's going on in this block (i.e. forcing delete of the empty body created above etc)?  Otherwise this if statement is a bit cryptic.